src/ui/components/Comments/TranscriptComments/CommentCard.tsx

```diff
@@ -87,6 +88,18 @@ function CommentCard({
     pendingComment?.comment.time === currentTime &&
     pendingComment?.comment.point === executionPoint;
 
+  const load = useDraftJS();
```

Comment:
You might move this to the top next to `useAuth0` so the hooks stay together which helps reduce the chances of accidentally bailing on render before a hook is called which makes React sad.

Reply:
fixed!